README.md

```diff
@@ -46,3 +46,20 @@ pd.plotting.register_matplotlib_converters()
 
 ## unit test status
 ![ci_workflow](https://github.com/unit8co/u8timeseries/workflows/ci_workflow/badge.svg)
+
```

Comment:
really cool, thanks for the documentation. We can keep it in the README for now, however at some point I think this should move elsewhere, as README will be meant for general public mostly.

Reply:
Could be actually moved to the top of version bumping script